app/services/analytics_events.rb

```diff
@@ -634,10 +634,6 @@ def idv_doc_auth_agreement_visited(**extra)
     track_event('IdV: doc auth agreement visited', **extra)
   end
 
-  def idv_doc_auth_cancel_link_sent_submitted(**extra)
```

Comment:
Are these analytics events re-implemented elsewhere or are they not relevant anymore?

Reply:
They were only used by the actions being deleted, and have been replaced in #8581 with additional params for hybrid_handoff_visited and document_capture_visited.